Handler/UploadStackage.hs

```diff
@@ -139,6 +139,7 @@ putUploadStackageR = do
                         , stackageDesc = "No description provided"
                         , stackageHasHaddocks = False
                         , stackageSlug = baseSlug
+                        , stackageGhcMajorVersion = Nothing -- Assumption: this file is deprecated
```

Comment:
Yes, this is mostly accurate. UploadStackage is still available for non-Nightly and non-LTS builds.